commands/enrich/enrich.go

```diff
@@ -0,0 +1,304 @@
+				if e.ExitCode() != coreutils.ExitCodeVulnerableBuild.Code {
+					err = errors.New("Scan command failed. " + err.Error())
+				}
+			}
+		}
+	}()
+	_, xrayVersion, err := xrutils.CreateXrayServiceManagerAndGetVersion(enrichCmd.serverDetails)
+	if err != nil {
+		return err
+	}
+
+	// Validate Xray minimum version for graph scan command
+	err = clientutils.ValidateMinimumVersion(clientutils.Xray, xrayVersion, scangraph.GraphScanMinXrayVersion)
+	if err != nil {
+		return err
+	}
+
+	log.Info("JFrog Xray version is:", xrayVersion)
+
+	threads := 1
+	if enrichCmd.threads > 1 {
+		threads = enrichCmd.threads
+	}
+
+	// resultsArr is a two-dimensional array. Each array in it contains a list of ScanResponses that were requested and collected by a specific thread.
+	resultsArr := make([][]*ScanInfo, threads)
+	fileProducerConsumer := parallel.NewRunner(enrichCmd.threads, 20000, false)
+	fileProducerErrors := make([][]formats.SimpleJsonError, threads)
+	indexedFileProducerConsumer := parallel.NewRunner(enrichCmd.threads, 20000, false)
+	indexedFileProducerErrors := make([][]formats.SimpleJsonError, threads)
+	fileCollectingErrorsQueue := clientutils.NewErrorsQueue(1)
+	// Start walking on the filesystem to "produce" files that match the given pattern
+	// while the consumer uses the indexer to index those files.
+	enrichCmd.prepareScanTasks(fileProducerConsumer, indexedFileProducerConsumer, resultsArr, indexedFileProducerErrors, fileCollectingErrorsQueue, xrayVersion)
+	enrichCmd.performScanTasks(fileProducerConsumer, indexedFileProducerConsumer)
+
+	// Handle results
+	flatResults := []xrutils.ScaScanResult{}
+	for _, arr := range resultsArr {
+		for _, res := range arr {
+			flatResults = append(flatResults, xrutils.ScaScanResult{Target: res.Target, XrayResults: []services.ScanResponse{*res.Result}})
+		}
+	}
+	if enrichCmd.progress != nil {
+		if err = enrichCmd.progress.Quit(); err != nil {
+			return err
+		}
+
+	}
+
+	fileCollectingErr := fileCollectingErrorsQueue.GetError()
+	var scanErrors []formats.SimpleJsonError
+	if fileCollectingErr != nil {
+		scanErrors = append(scanErrors, formats.SimpleJsonError{ErrorMessage: fileCollectingErr.Error()})
+	}
+	scanErrors = appendErrorSlice(scanErrors, fileProducerErrors)
+	scanErrors = appendErrorSlice(scanErrors, indexedFileProducerErrors)
+
+	scanResults := xrutils.NewAuditResults()
+	scanResults.XrayVersion = xrayVersion
+	scanResults.ScaResults = flatResults
+
+	isxml, err := isXML(scanResults.ScaResults)
+	if err != nil {
+		return
+	}
+	if isxml {
+		if err = xrutils.NewResultsWriter(scanResults).
+			AppendVulnsToXML(); err != nil {
+			return
+		}
+	} else {
+		if err = xrutils.NewResultsWriter(scanResults).
+			AppendVulnsToJson(); err != nil {
+			return
+		}
+	}
+
+	if err != nil {
+		return err
+	}
+
+	if err = utils.RecordSecurityCommandOutput(utils.ScanCommandSummaryResult{Results: scanResults.GetSummary(), Section: utils.Binary}); err != nil {
+		return err
+	}
```

Comment:
```suggestion
```
This method will create an entry at the `Binary` tab in the Job summary (Github actions).
Are you sure you want to add this? you can try running a command with `"JFROG_CLI_COMMAND_SUMMARY_OUTPUT_DIR"` env var pointing to a dir. after running a command it will generate a `.md` file with the job summary in it.

Reply:
I don't need it